packages/go/analysis/ad/ad.go

```diff
@@ -905,21 +905,20 @@ func GetADCSESC6aEdgeComposition(ctx context.Context, db graph.Database, edge *g
 
 		}
 
-		return true, nil
-	}); err != nil {
-		return paths, err
+		return true
+	})
+
+	if closureErr != nil {
+		return paths, closureErr
 	}
```

Comment:
Make sure we're actually consuming the closureErr and replicating the original logic